vtex-apps/store-newsletter
# First working implementation

#### What problem is this solving?

Adds the first working implementation of the following blocks:

- `newsletter-form`
- `newsletter-input-email`
- `newsletter-input-name`
- `newsletter-input-phone`
- `newsletter-checkbox-confirmation`
- `newsletter-submit`

#### How to test it?

You can go to [Workspace](https://victormiranda--storecomponents.myvtex.com/) and scroll to the bottom of the page, you should see the newsletter form there. (It doesn't look good, but that's kind of intentional, we could make it look good using CSS).

You can also go to the [Site Editor](https://victormiranda--storecomponents.myvtex.com/admin/cms/site-editor) and see edit it there.

To read the rendered documentation, go to https://github.com/vtex-apps/store-newsletter/blob/feature/first-implementation/docs/README.md.

#### Screenshots or example usage:

![Screen Shot 2020-10-08 at 14 43 35](https://user-images.githubusercontent.com/27777263/95494699-ad1ac080-0974-11eb-87cc-6ccee74457ad.png)

#### Describe alternatives you've considered, if any.



#### Related to / Depends on

Depends on https://github.com/vtex-apps/store-graphql/pull/500.

#### How does this PR make you feel? [:link:](http://giphy.com/)

I don't know about the "clean" part... but hey, it's fresh 😂 

![](https://media.giphy.com/media/kDBhX1Il2PZL66ljiL/giphy.gif)
